DragaliaAPI/Models/AutoMapper/HelperMapProfile.cs

```diff
@@ -0,0 +1,68 @@
+﻿using AutoMapper;
+using DragaliaAPI.Models.Generated;
+using DragaliaAPI.Shared.Definitions.Enums;
+
+namespace DragaliaAPI.Models.AutoMapper;
+
+public class HelperMapProfile : Profile
+{
+    public HelperMapProfile()
+    {
+        this.CreateMap<AtgenSupportChara, CharaList>()
+            .ForMember(x => x.exp, opts => opts.Ignore())
+            .ForMember(x => x.is_new, opts => opts.Ignore())
+            .ForMember(x => x.burst_attack_level, opts => opts.Ignore())
+            .ForMember(x => x.limit_break_count, opts => opts.Ignore())
+            .ForMember(x => x.combo_buildup_count, opts => opts.Ignore())
+            .ForMember(x => x.gettime, opts => opts.Ignore())
+            .ForMember(x => x.mana_circle_piece_id_list, opts => opts.Ignore())
```

Comment:
dragons and prints likely aren't necessary to make detailed since the only stuff that really gets left out is things like is_lock and is_new which aren't relevant for helpers
could try to define characters as CharaList tho

Reply:
actually the more i think about it the harder i find myself able to wrap my head around changing it to be that way
like when you're selecting a helper the game only requests the summarized version with a list of all helpers, which is QuestGetSupportUserListData
and when it asks for the full character data when starting a quest with a helper, the mana nodes don't actually matter bc the boosted stats and skill/ability levels from mana nodes have already taken effect and are reflected in the character's stats regardless of what the mana nodes are listed as
the only thing that truly gets affected is when checking the helper's setup when on the screen to choose a helper, the mana nodes don't reflect what they actually are (ie. your lvl 10 dy!ilia probably doesn't have the stats of a 50mc dy!ilia even though she's listed as 50mc but also i inputted everything else at max mc stats so those would be the correct mc counts) but if/when we implement being able to set your own helper then HelperService would prob make a request to like UnitRepository to get the mana nodes of the character matching the viewer id and chara id
but then that's DbPlayerCharaData and i think i would die if i had to make a byte cast over like every field when converting it all

i have no idea if this made any sense at all
tl;dr i think it's kinda too much effort to try and convert everything (esp w/ like, 2000 lines worth of unit data rn) for what's essentially just a visual discrepancy on a single helper

Reply:
> and when it asks for the full character data when starting a quest with a helper, the mana nodes don't actually matter bc the boosted stats and skill/ability levels from mana nodes have already taken effect and are reflected in the character's stats regardless of what the mana nodes are listed as

to be honest, as long as it's functionally the same it's fine. Once we replace this with real DB entries this can be fixed